src/SafeCurl.php

```diff
@@ -171,6 +185,33 @@ public function setFollowLocationLimit(int $limit): void {
         $this->followLocationLimit = $limit;
     }
 
+    /**
+     * After a host's IPs have been resolved, we set them as a cURL option.
+     * This prevents the use of DNS rebinding as an SSRF attack
+     *
+     * @param array $url
+     */
+    public function setHostIPs(array $url): void {
```

Comment:
I know this is easier to test as a public member, but it seems like something we don't want to expose (it'll be a pain to enforce `$url` is actually the return value of `validateUrl` if used outside this class). Can it be defined as a private member, while still being testable?

Reply:
Yup, just used reflection in the test to open up the method when testing.